### ISSUE TITLE
Type 'SetState<StoreState>' is not assignable to type 'SetState<Record<string | number | symbol, any>>'.

### ISSUE THREAD

**Issue by Rhymond** [beerose/simple-zustand-devtools#22]
When I try to use devtools in my typescript application, returned value from `create()`  is not assignable to `mountStoreDevtool` store argument. Error:
```
Type 'SetState<StoreState>' is not assignable to type 'SetState<Record<string | number | symbol, any>>'.
```

```
export const useStore = create<StoreState>((set, get) => ({
   ...
})); 
// returns StoreApi<StoreState>

mountStoreDevtool("Store", useStore); 
// expected useStore of type StoreApi<Record<string | number | symbol, any>>;
```

Is there a way to fix it?

**Comment by timmeade** [comment]
Same here

```
Argument of type 'UseBoundStore<ProfileStore, StoreApi<ProfileStore>>' is not assignable to parameter of type 'ZustandStore'.
  Types of property 'setState' are incompatible.
```

**Comment by beerose** [comment]
Thanks for the report!

Should be fixed in https://github.com/beerose/simple-zustand-devtools/commit/867096a404a3e6ed3245875ff0a4df5a9d94c240 — released as 1.1.0.
